src/QirRuntime/test/QIR-static/qir-test-math.cpp

```diff
@@ -23,3 +28,96 @@ TEST_CASE("QIR: Math.ArcTan2", "[qir.math][qir.Math.ArcTan2]")
     REQUIRE(0 == Microsoft__Quantum__Testing__QIR__Math__ArcTan2Test__body());
 }
 
+TEST_CASE("QIR: Math.DrawRandomInt", "[qir.math][qir.Math.DrawRandomInt]")
+{
+#ifdef __APPLE__
+    std::vector<int64_t> expectedLargeNumbers( {  
```

Comment:
I'm not convinced testing the specific output from the random generator, even if we seed it, is a good idea. We are already seeing differences between platforms. What if there are differences between clang versions? Will we have to update the tests after moving to 11.0?

Our implementation is a thin wrapper around std random generator, why are we testing the specifics of that generator?

Reply:
Agreed. Changing.